packages/wdio-shared-store-service/shared-store-service.d.ts

```diff
@@ -1,5 +1,5 @@
 declare namespace WebdriverIO {
-    interface BrowserObject {
+    interface Browser {
```

Comment:
As a rule of thumb, if we need to extend core properties, we should extend `Browser` interface. This is because `BrowserObject` can extends `ClientSync` or `ClientAsync`, so if we modify `BrowserObject`, there will be duplicate codes everywhere